packages/auth/src/tests/signature.ts

```diff
@@ -69,18 +69,33 @@ export async function generateSigHeaders({
     body?: unknown
     authorization?: string
   }
-}): Promise<{ sigInput: string; signature: string; contentDigest?: string }> {
```

Comment:
Are we going to be able to just replace this with `http-signature-utils`' `createSignatureHeaders`?
- https://github.com/interledger/rafiki/pull/797

Reply:
Yeah, pretty easily I think.